src/state/electrumX/sagas.ts

```diff
@@ -241,8 +241,11 @@ export function* listenToInternetConnection() {
   const chan: unknown = yield call(emitInternetConnectionChange);
 
   while (true) {
+    yield delay(2000); // In case connection with electrum lost we should wait to reconnect before showing alert;
```

Comment:
little not feel good with this type solution

Reply:
fixed